isValueEven.js

```diff
@@ -1,7 +1,30 @@
-function isEvenValue(number) {
-  if (number / 2 === 1) {
+function isValueEven(number) {
+  isInputNumber(number);
+  isInputAnInteger(number);
+  isInputZero(number);
+  if (number % 2 === 0) {
     return true;
-  } else {
-    return false;
   }
+  return false;
 }
+
+isInputNumber = (input) => {
+  if (typeof input !== "number" || isNaN(input) === true) {
+    throw new Error("You must type number");
+  }
+};
+
+isInputAnInteger = (input) => {
+  if (Number.isInteger(input) === false) {
+    throw new Error("You must not type an integer");
+  }
+};
+
+isInputZero = (input) => {
+  if (input === 0) {
+    throw new Error("You must not type 0");
+  }
+};
```

Comment:
We still can check if 0 is even or not :) So this error throw is unnecessary

Reply:
validation deleted